lib/tags/profile-settings.tag

```diff
@@ -360,6 +365,7 @@
                     let successMessage = i18n.emailSuccessMessage;
                     self.refs.email.classList.remove('graphjs-error');
                     self.refs.email.classList.add('graphjs-success');
+                    self.refs.email.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.email refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above